### PR TITLE
Cherry pick LKG/MsUCRT fixes

### PR DESCRIPTION
**Pull request by Scottj1s**
See:
https://github.com/microsoft/WindowsAppSDK/pull/4933

### CODE REVIEW

**Review by alexlamtest — APPROVED** [review]

File: installer/dev/main.cpp [hunk 1]
```diff
@@ -6,6 +6,11 @@
 #include "install.h"
 #include "resource.h"
 
+#ifndef GUID_NULL
+struct __declspec(uuid("00000000-0000-0000-0000-000000000000")) GUID_NULL;
+#define GUID_NULL __uuidof(struct GUID_NULL)
+#endif
```
Comment (DrusTheAxe) [inline installer/dev/main.cpp:12]: Why not

```
constexpr GUID GUID_NULL{};
```
?

**Review by DrusTheAxe — APPROVED** [review]
LGTM modulo comments

File: installer/dev/main.cpp (hunk 1 above)
Comment (Scottj1s) [reply installer/dev/main.cpp:12]: no good reason - lifted the pattern from OS code

**Pull request merged**